tests/Rules/UseTypeHintForNewMethodsRuleTest.php

```diff
@@ -55,5 +55,6 @@ public function testRule(): void
         $this->analyse([$dataDirectory . 'MethodFooEThreeParametersWithIdentifierTypeHint.php'], []);
         $this->analyse([$dataDirectory . 'MethodFooGWithoutTypeHintButExcluded.php'], []);
         $this->analyse([$dataDirectory . 'MethodFooHWithoutTypeHintButExtendsAnExcludedClass.php'], []);
+        $this->analyse([$dataDirectory . 'JWithConstructAndGetSet.php'], []);
```

Comment:
Why `J`? :thinking:

Reply:
Initially I had sample test files `A.php`, `B.php`, `C.php`.

Then I went for improved file names such as `MethodFooHWithoutTypeHintButExtendsAnExcludedClass.php` but I continue to add one alphabet letter in each file.

Indeed maybe it's completely useless 😁

Reply:
It's confusing :sweat_smile:

Reply:
Replaced by "ClassWithConstructAndGetSet.php" in https://github.com/PrestaShop/phpstan-prestashop/pull/17/commits/5501859a03131b791d9cc8e70dc9d57c8f2d5e81